strings.txt

```diff
@@ -8925,7 +8927,7 @@
     hu = %@ MB-t kell letöltened, de csak %@ MB hely szabad.
     el = Πρέπει να κάνετε λήψη %@ MB, αλλά μόνο %@ MB είναι διαθέσιμα.
     ar = تحتاج إلى تحميل %@ ميجابايت، ولكن لا يتوافر سوى %@ ميجا بايت.
-    de = Der Download erfordert %@ MB, es sind jedoch nur %@ MB verfügbar.
+    de = Sie müssen %@ MB herunterladen, aber es sind nur %@ MB verfügbar.
```

Comment:
This one should be improved. The new one sounds as if the download does not provide enough MB. Maybe better:
Sie wollen %@ MB herunterladen, der Speicherplatz reicht aber nur für %@ MB.